sw/airborne/firmwares/rotorcraft/autopilot_arming_yaw.h

```diff
@@ -71,8 +74,9 @@ static inline void autopilot_arming_set(bool motors_on)
 {
   if (motors_on) {
     autopilot_check_motor_status = STATUS_MOTORS_ON;
-  } else {
+  } else if (autopilot_check_motor_status == STATUS_MOTORS_ON) {
```

Comment:
this is just in case it is called repeatedly?

Reply:
The 'difficulty' here is that from autopilot_check_motor_status==STATUS_MOTORS_AUTOMATICALLY_OFF you can now do a rapid re-arm. If you would kill from being in a disarmed state and then immediately unkill, you could otherwise arm the vehicle. This way, only if you kill from an motors_on state you can rapidly re-arm.